examples/gitlab/gitlab.yml

```diff
@@ -0,0 +1,134 @@
+name: gitlab
+labels:
+    app: gitlab
+replicas: 1
+containers:
+- name: gitlab
+  image: gitlab/gitlab-ce:9.4.1-ce.0
+  imagePullPolicy: ""
+  env:
+  ## General GitLab Configs
+  ##
+  # This is a free-form env var that GitLab Omnibus uses to configure
+  # everything. We're passing this in from a configmap and pulling some
+  # of the values from the env vars defined below. This is done to
+  # avoid leaving secrets visible in kubectl.
+  - name: GITLAB_OMNIBUS_CONFIG
+    valueFrom:
+      configMapKeyRef:
+        name: gitlab
+        key: gitlab_omnibus_config
+  - name: GITLAB_ROOT_PASSWORD
+  - name: EXTERNAL_URL
+    value: "http://your-domain.com/"
+  ## DB configuration
+  ##
+  - name: DB_HOST
+    value: postgresql
+  - name: DB_USER
+    valueFrom:
+      secretKeyRef:
+        name: gitlab
+        key: db-user
+  - name: DB_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: gitlab
+        key: db-password
+  - name: DB_DATABASE
+    value: "gitlab"
+  ## Redis configuration
+  ##
+  - name: REDIS_HOST
+    value: redis
+  - name: REDIS_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: gitlab
+        key: redis-password
+  livenessProbe:
+    httpGet:
+      path: /help
+      port: http
+    # This pod takes a very long time to start up. Be cautious when
+    # lowering this value to avoid Pod death during startup.
+    initialDelaySeconds: 200
+    timeoutSeconds: 1
+    periodSeconds: 10
+    successThreshold: 1
+    failureThreshold: 10
+  readinessProbe:
+    httpGet:
+      path: /help
+      port: http
```

Comment:
also here port needs to be 80

Reply:
Like it was shown in the diff here: https://github.com/kedgeproject/kedge/pull/170#discussion_r136240919